asg-unreg-nodes.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+"""
+asg-unreg-nodes.py
+List autos-scaled nodes that failed to register with Yarn Resource Manager
+Sample config
+{
+    "access_key": "key",
+    "secret_key": "secret",
+    "group_name": "datapipeline-in-stag-spot-asg"
+
+    # optional: depending on running on the master/any node
+    "HOSTNAME": "datapipeline-in-master.stag.indix.tv"
+}
+"""
+import boto
+import boto.ec2.autoscale
+from boto.ec2.autoscale import AutoScaleConnection
+import requests
+import json
+import socket
+
+config = json.load(open("config.json"))
+access_key = config['access_key']
```

Comment:
Can we please not take in AWS Keys from a config? It becomes messy on how we would pass them in a flat file. Please let's assume the instance where we run the script has IAM roles configured to query EC2 APIs.